src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/contentgrid/view/ContentGrid.kt

```diff
@@ -0,0 +1,72 @@
+package org.wycliffeassociates.otter.jvm.app.ui.contentgrid.view
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.Property
+import javafx.scene.layout.Priority
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Content
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.contentgrid.viewmodel.ContentGridViewModel
+import org.wycliffeassociates.otter.jvm.app.widgets.card.DefaultStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.card.card
+import tornadofx.*
+
+
+class ContentGrid : Fragment() {
+    private val viewModel: ContentGridViewModel by inject()
+
+    val activeCollection: Property<Collection> = viewModel.activeCollectionProperty
+    val activeProject: Property<Collection> = viewModel.activeProjectProperty
+    val activeContent: Property<Content> = viewModel.activeContentProperty
+
+    init {
+        importStylesheet<ContentGridStyles>()
+        importStylesheet<DefaultStyles>()
+    }
+
+    override val root = hbox {
+        hgrow = Priority.ALWAYS
+        vgrow = Priority.ALWAYS
+        addClass(AppStyles.appBackground)
+        addClass(ContentGridStyles.panelStyle)
+        vbox {
```

Comment:
if the only thing in the hbox is this vbox, can we just replace the hbox with this vbox?

nested layouts affect performance a lot